profile/README.md

```diff
@@ -0,0 +1,90 @@
+# Rust Bitcoin Community
+
+A Series of projects to implement various Bitcoin protocols in Rust.
+
+## Core Crates
+
+The "core" crates developed and maintained collectively by the `rust-bitcoin` devs.
+
+For these crates we hold ourselves to high standards relating to security, testing, release
+schedule, API design, and coding style. Our current focus is on stabilising these crates [0].
```

Comment:
This `[0]` footnote is broken.

```suggestion
schedule, API design, and coding style. Our current focus is on stabilising these crates[^0].
```

Ref: https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/

Reply:
Also down below we say `stabilize`, so maybe we should also use the american "way" here:

```suggestion
schedule, API design, and coding style. Our current focus is on stabilizing these crates [0].
```

Reply:
We also have the spelling `stabalize` (which @tcharding has very consistently used so I assume it's a dialect thing). I think we should standardize on `stabilize` and `stabilizing`.

Reply:
yeah I have my editor set to UK English and manually use US English when working, perhaps I have to bight the bullet and just use US English in all my life - but that hurts a bit.

Reply:
Yeah, as a Canadian I feel you. I choose instead to fill my brain with Americanisms and then do them all by memory (and at work we employ Americans to further correct my grammar).